app/pages/collections/[suffix]/admin.tsx

```diff
@@ -0,0 +1,417 @@
+                refetchFn={refetchFn}
+                key={`submission-${index}`}
+              />
+            )
+          })}
+        </>
+      )}
+      {pendingSubmissions?.submissions.length! === 0 && collection?.type.type === "COMMUNITY" && (
+        <div className="">
+          <div className="relative flex w-full flex-grow flex-col rounded-lg border-2 border-dashed border-gray-800 text-center focus:outline-none focus:ring-2 focus:ring-indigo-500  focus:ring-offset-2 dark:border-white">
+            <div className="table h-full w-full flex-grow">
+              <div className="h-28 w-1/4 sm:table-cell"></div>
+              <span className="mx-auto table-cell align-middle text-sm font-medium leading-4">
+                <>
+                  <div className="">No pending submissions. Maybe request some?</div>
+                </>
+              </span>
+              <div className="hidden w-1/4 sm:table-cell"></div>
+            </div>
+          </div>
+        </div>
+      )}
+    </div>
+  )
+}
+
+const CollectedWorks = ({ collection, editorIdSelf, refetchFn, editorIsAdmin }) => {
+  const [addWorkMutation] = useMutation(addWork)
+  const [createReferenceMutation] = useMutation(createReferenceModule)
+
+  return (
+    <div className="mx-4 my-8 xl:mx-0">
+      <h2 className="my-4 text-xl">Collected works</h2>
+      <div>
+        <Autocomplete
+          className=""
+          openOnFocus={false}
+          defaultActiveItemId="0"
+          getSources={({ query }) => [
+            {
+              sourceId: "products",
+              async onSelect(params) {
+                const { item, setQuery } = params
+                toast.promise(
+                  addWorkMutation({
+                    collectionId: collection!.id,
+                    editorId: editorIdSelf,
+                    moduleId: parseInt(item.objectID),
+                  }),
+                  {
+                    loading: "Adding work to collection...",
+                    success: () => {
+                      refetchFn()
+                      return "Added work to collection!"
+                    },
+                    error: "Failed to add work to collection...",
+                  }
+                )
+              },
+              getItems() {
+                return getAlgoliaResults({
+                  searchClient,
+                  queries: [
+                    {
+                      indexName: `${process.env.ALGOLIA_PREFIX}_modules`,
+                      query,
+                    },
+                  ],
+                })
+              },
+              templates: {
+                item({ item, components }) {
+                  return (
+                    <>
+                      {item.__autocomplete_indexName.match(/_modules/g) ? (
+                        <SearchResultModule item={item} />
+                      ) : (
+                        ""
+                      )}
```

Comment:
Would using the short-circuit operator, instead of the ternary, work the same way?

```suggestion
                      {item.__autocomplete_indexName.match(/_modules/g)  && 
                        <SearchResultModule item={item} />
```

Reply:
We could I think - we need to do a cleanup of this throughout the app and make sure to test. This is noted in #550 - @nasainsbury mentioned willing to work on this - maybe reach out to them on Discord?

Reply:
Sounds good. We have an issue to track this, so I'm going to add it there too: https://github.com/libscie/ResearchEquals.com/issues/546